### ISSUE TITLE
ops: old docker whois container remains running after new deploy

### ISSUE THREAD

**Issue by ccaputo** [peeringdb/peeringdb#806]
This may be a null issue once https://github.com/peeringdb/peeringdb/issues/701 is implemented, but for now it is a problem.  For example:

```
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
6162072bb6f8        pdb:server-latest   "./entrypoint.sh ine…"   41 hours ago        Up 41 hours                             condescending_wilbur
8ad9556e35d2        pdb:server-latest   "./entrypoint.sh uws…"   41 hours ago        Up 41 hours                             peaceful_taussig
f4604f4015ae        pdb:server-latest   "./entrypoint.sh ine…"   5 days ago          Up 5 days                               sad_swanson
b4b914f705c6        586b3c2d69d9        "./entrypoint.sh ine…"   2 weeks ago         Up 2 weeks                              zealous_shockley
```

**Comment by grizz** [comment]
Will be fixed with #701 -- the prod deploy steps have been changed for OS updates and reboots after code deploy, but we'll have to remember to check that during beta deploys.

**Comment by ccaputo** [comment]
Closing based on the upcoming retirement of the whois service: https://docs.peeringdb.com/blog/whois_to_close/
